src/app/CommandHandler.cpp

```diff
@@ -944,7 +944,8 @@ CHIP_ERROR TestOnlyExtractCommandPathFromNextInvokeRequest(TLV::TLVReader & invo
     case CommandHandler::NlFaultInjectionType::SkipSecondResponse:
         return "Single InvokeResponseMessages. Dropping response to second request";
     }
-    VerifyOrDieWithMsg(false, DataManagement, "TH Failure: Unexpected fault type");
+    ChipLogError(DataManagement, "TH Failure: Unexpected fault type");
+    chipAbort();
```

Comment:
Yes, this will also work, but why adding return if we are not returning but aborting? Abort is  marked as no return function. Also, why we are verifying that false is not true? IMHO using VerifyOrDieWithMsg is in fact obfuscation here.